tls/s2n_resume.c

```diff
@@ -972,6 +973,14 @@ int s2n_connection_add_new_tickets_to_send(struct s2n_connection *conn, uint8_t
     return S2N_SUCCESS;
 }
 
+extern int s2n_connection_get_tickets_sent(struct s2n_connection *conn, uint16_t *num)
```

Comment:
Why is this extern? I thought only function declarations in header files were marked with extern.